libs/brain/tabs/src/lib/brn-tabs-trigger.ts

```diff
@@ -18,7 +18,7 @@ import { BrnTabs } from './brn-tabs';
 		'[id]': '_labelId()',
 		type: 'button',
 		role: 'tab',
-		'[tabindex]': 'selected() ? "0": "-1"',
+		'[tabindex]': '_disabled() ? "-1" : (selected() ? "0" : "-1")',
```

Comment:
I think technically this should be a number, as we are binding to the tabindex property, not the tabindex attribute.

```suggestion
		'[tabindex]': '_disabled() ? -1 : (selected() ? 0 : -1)',
```